### PR TITLE
Add Anycoin Direct

### PR DESCRIPTION
**Pull request by mxxcon**

### CODE REVIEW

**Comment by mxxcon** [comment]
@2factorauth/collaborators bump?

File: _data/cryptocurrencies.yml [hunk 1]
```diff
@@ -6,6 +6,13 @@ websites:
       software: Yes
       doc: https://anxpro.com/pages/security
 
+    - name: AnyCoin Direct
```
Comment (Jawshy) [inline _data/cryptocurrencies.yml]: The capitalisation of their name appears to be `Anycoin` not `AnyCoin`.

File: _data/cryptocurrencies.yml [hunk 2]
```diff
@@ -6,6 +6,13 @@ websites:
...
+      url: https://anycoindirect.eu
+      img: anycoindirect.png
+      tfa: Yes
+      software: Yes
+      doc: https://anycoindirect.eu/de/account/twofactorauthentication
```
Comment (Jawshy) [inline _data/cryptocurrencies.yml:14]: This link redirects me to a login page. I guess that there is no public documentation.

**Review by Jawshy — CHANGES_REQUESTED** [review]
`name` and `doc`.

File: _data/cryptocurrencies.yml (hunk 2 above)
Comment (mxxcon) [reply _data/cryptocurrencies.yml:14]: Probably. This info was provided to us through the submission form.

**Review by Jawshy — APPROVED** [review]
I confirmed Anycoin Direct's support for software-based two-factor authentication via email correspondence to their support email address.

File: _data/cryptocurrencies.yml (hunk 2 above)
Comment (jamcat22) [reply _data/cryptocurrencies.yml:14]: Since the doc link isn't public, should it be removed?

**Comment by mxxcon** [comment]
@2factorauth/collaborators bump...

**Comment by mxxcon** [comment]
Waited long enough.

**Pull request closed without merging**